src/main/java/io/omnition/loadgenerator/util/OpenTracingTraceConverter.java

```diff
@@ -20,24 +20,12 @@ private OpenTracingTraceConverter() {}
             otSpanBuilder = addModelTag(tag, otSpanBuilder);
         }
         final Tracer.SpanBuilder finalSpanBuilder = otSpanBuilder;
-        span.refs.forEach(ref -> {
-            switch (ref.refType) {
-                case CHILD_OF:
-                    finalSpanBuilder.addReference(
-                            io.opentracing.References.CHILD_OF,
-                            parentSpans.get(ref.fromSpanId).context()
-                    );
-                    break;
-                case FOLLOWS_FROM:
```

Comment:
From what I understand FOLLOWS_FROM is not used anywhere in this generator, so this is fine, right?